storage/storage_tests/storage_tests.cpp

```diff
@@ -1789,5 +1795,31 @@ UNIT_TEST(StorageTest_GetTopmostNodesForWithLevel)
   TEST_EQUAL(path[0], "France_Burgundy_Saone-et-Loire", ());
 }
 
+UNIT_TEST(StorageTest_FalsePolicy)
+{
+  DummyDownloadingPolicy policy;
+  Storage storage;
+  storage.SetDownloadingPolicy(&policy);
+
+  storage.Init(&OnCountryDownloaded /* didDownload */,
+               [](TCountryId const &, TLocalFilePtr const) { return false; } /* willDelete */);
+  storage.SetDownloaderForTesting(make_unique<TestMapFilesDownloader>());
+  storage.SetCurrentDataVersionForTesting(1234);
+
+  auto const countryId = storage.FindCountryIdByFile("Uruguay");
+  auto const countryFile = storage.GetCountryFile(countryId);
 
+  // To prevent interference from other tests and on other tests it's
```

Comment:
interference with other tests?
influence on and from other tests?

Reply:
"interference" SGTM